automated-comments/go/two-fer/strings_join_used_for_concatenation.md

```diff
@@ -1,2 +1,3 @@
-Don't use `strings.Join` if the data is not of type `[]string`.
-Use `fmt.Sprintf` instead. It's a very powerful way of building up strings from multiple pieces.
+`strings.Join` is only normally used if the data is of type `[]string`.
```

Comment:
See comment below about being Dutch, but I like the below:

```suggestion
`strings.Join` is usually only used if the data is of type `[]string`.
```

Reply:
It is "more common" than what?  Perhaps "it is common" is better here.